src/Nest/Analysis/Plugins/Icu/Collation/IcuCollationCaseFirst.cs

```diff
@@ -5,9 +5,12 @@
 namespace Nest
 {
 	/// <summary>
-	/// Sets the alternate handling for strength quaternary to be either shifted or non-ignorable.
-	/// Which boils down to ignoring punctuation and whitespace.
+	/// Controls which case is sorted first when case is not ignored for
```

Comment:
Quite the comment change!

Reply:
The summary was incorrect; was copied from `IcuCollationAlternate`